### PR TITLE
Explore: Remove emotion error when displaying logs

### LINKED ISSUE #22645: Interpolation of className coming from emotion css prop is deprecated.
**What happened**:

A number of lines such as:
```
console.error node_modules/@emotion/serialize/dist/serialize.cjs.dev.js:180
>>       Interpolating a className from css`` is not recommended and will cause problems with composition.
>>       Interpolating a className from css`` will be completely unsupported in a future major version of Emotion
```
are being logged during tests/build.

**What you expected to happen**:

No emotion errors in Grafana repo.

**How to reproduce it (as minimally and precisely as possible)**:

Fetch a devenv and run `yarn test`. Look at components using emotion that do something with innerElement on hover over the outerElement, like tooltips.

Minimal generic example to show what we are talking about:
```jsx
const innerComponent = css`
  display: none;
`;

const outerComponent = css`
  &:hover .${innerComponent} {
    display: initial;
  }
`;

<div className={parent}>
  <div className={child} />
</div>
```
also mentioned there: https://github.com/emotion-js/emotion/issues/1217

### PR DESCRIPTION
**Pull request by ivanahuckova**
**What this PR does / why we need it**:
Removes `Interpolating a className from css`` is not recommended` error. We were interpolating a className `context`, but this isn't necessary as we can just use the non-interpolated-classname and not have this error behaviour. 
image

Before: 
image

After (still works the same):
image

Fixes: https://github.com/grafana/grafana/issues/22645

### CODE REVIEW

File: packages/grafana-ui/src/components/Logs/getLogRowStyles.ts [hunk 1]
```diff
@@ -69,7 +69,7 @@ export const getLogRowStyles = stylesFactory((theme: GrafanaTheme, logLevel?: Lo
       vertical-align: top;
 
       &:hover {
-        .${context} {
+        .log-row-context {
```
Comment (zoltanbedi) [inline packages/grafana-ui/src/components/Logs/getLogRowStyles.ts:72]: You can move the variable context to line 64 then, can't you?

Comment (ivanahuckova) [reply packages/grafana-ui/src/components/Logs/getLogRowStyles.ts:72]: Yes, good point! On it.

Comment (ivanahuckova) [reply packages/grafana-ui/src/components/Logs/getLogRowStyles.ts:70]: You meant it like this, right? https://github.com/grafana/grafana/pull/31026/commits/487b04e09ed602cf923ea13165e9a4f5f1757b38

Comment (zoltanbedi) [reply packages/grafana-ui/src/components/Logs/getLogRowStyles.ts:70]: yes

**Review by zoltanbedi — APPROVED** [review]
lgtm 🌸  so wanted to see this fixed for a long time

**Pull request merged**